api/base/serializers.py

```diff
@@ -13,6 +12,27 @@
 from api.base.exceptions import InvalidQueryStringError, Conflict
 
 
+def format_relationship_links(related_link, self_link, rel_meta, self_meta):
+
+    ret = {
+        'links': {
+            'related': {
+                'href': related_link,
+                'meta': rel_meta
+            },
+            'self': {
+                'href': self_link,
+                'meta': self_meta
+            }
+        }
+    }
+
+    # Not currently returning self links
+    if not ret['links']['self']['href']:
```

Comment:
I will add 
```
if not ret['links']['related']['href']:
    del ret['links']['related']
```

So this makes empty links dicts, but Sam's PR will address this?

![screen shot 2015-11-02 at 1 48 34 pm](https://cloud.githubusercontent.com/assets/9755598/10890828/56cfaf40-8168-11e5-84ec-49fca93e12bd.png)